### DISCUSSION TITLE
Remote desktop maximized on multiple monitors

### DISCUSSION THREAD

**Discussion by Demon1986** [kangyu-california/PersistentWindows#183] [Q&A] [answered]
at work i run dual screens maximized. i tried this to help with the window going to "Restored" when the monitor comes back from sleep.
however, it doesn't seem to do as i had expected, and there doesn't seem to be any other shortcuts for capture, other than click on the icon, wich only works when the session is minimized, or in a restored state.

how the hell do i solve this?

**Comment by kangyu-california** [comment, marked as answer]
please follow (https://github.com/kangyu-california/PersistentWindows/issues/184) for solution

**Reply by Demon1986** [reply]
how would i get that "fix" here? as it doesns't seem to work on the current release.

**Reply by kangyu-california** [reply]
Attached is a patch fix which allows user to modify the capture command from instant to delayed by holding the SHIFT key.
With delayed capture, user has 5 seconds to make a pose of window layout before the capture is actually taken.
The 5-seconds delay can be modified from command line, for example -delay_capture 10

[PersistentWindows5.39_patch183.zip](https://github.com/kangyu-california/PersistentWindows/files/9686675/PersistentWindows5.39_patch0.zip)
